### PR TITLE
feat(killswitch): Move killswitch functionality into here

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -5,20 +5,30 @@ inputs:
...
+      run: |
+        if [[ -z $INPUT_FORCE && curl -s "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=release-blocker" | grep -Pzvo '\[[\s\n\r]*\]' ]]; then
+          echo "Open release-blocking issues found, cancelling release...";
+          curl -sf -X POST -H 'Accept: application/vnd.github.v3+json' -H 'Authorization: token ${{ github.token }}' https://api.github.com/repos/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID/cancel;
+        fi
     - name: Determine version
       run: |
-        if [[ -n '${{ inputs.version }}' ]]; then
-        echo 'RELEASE_VERSION=${{ inputs.version }}' >> $GITHUB_ENV;
+        if [[ -n "$INPUT_VERSION" ]]; then
```
Comment (chadwhitacre) [inline action.yml:23]: Why is this changing with this PR?

**Review by chadwhitacre — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (BYK) [reply action.yml:23]: Just aligning everything to use env variables if possible as I was at it. Agree that it is less than ideal but I think we can accept it for now?

Comment (chadwhitacre) [reply action.yml:23]: Okay then why do we still have `version` in inputs above?

**Review by chadwhitacre — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (BYK) [reply action.yml:23]: >Okay then why do we still have version in inputs above?

Those definitions are what makes the `INPUT_*` env vars available/set:

https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#example
>When you specify an input to an action in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name INPUT_<VARIABLE_NAME>. The environment variable created converts input names to uppercase letters and replaces spaces with _ characters.

Comment (chadwhitacre) [reply action.yml:23]: Perfect, thanks. 👍

**Review by chadwhitacre — APPROVED** [review]

**Pull request merged**